CONTRIBUTING.md

```diff
@@ -155,14 +155,14 @@ currently using a minimum version of PyCharm 2019.2.4.
     `python -m eland.tests.setup_tests` to setup test environment -*note
     this modifies Elasticsearch indices*
 -   Install local `eland` module (required to execute notebook tests)
-    `python install setup.py`
+    `python setup.py install`
 -   Run `pytest --nbval  --doctest-modules` to validate install
 -   To test specific versions of Python use `nox -s test-3.8`
 -   To run the automatic formatter and check for lint issues
     run `nox -s blacken`
 
 
 ### Documentation
-
+-   Install pandoc `sudo apt-get install -y pandoc`
```

Comment:
Let's amend this step with something like:

- [Install Pandoc on your system](https://pandoc.org/installing.html)
  For Ubuntu or Debian it is `sudo apt-get install -y pandoc`